superset/sql_parse.py

```diff
@@ -272,7 +272,7 @@ def _extract_tables_from_sql(self) -> set[Table]:
             logger.warning("Unable to parse SQL (%s): %s", self._dialect, self.sql)
             raise SupersetSecurityException(
                 SupersetError(
-                    error_type=SupersetErrorType.QUERY_SECURITY_ACCESS_ERROR,
+                    error_type=SupersetErrorType.INVALID_SQL_ERROR,
```

Comment:
Thanks! The intent here was that we often need to extract tables in order to see if a query can run, but obviously we do for other reasons. Potentially we could add a try/except in the security manager to capture this error and convert it to a security error, but I don't think it's necessary.